kafka-avro-adapter/src/common.h

```diff
@@ -74,6 +74,7 @@ struct Options
     std::string config = "Columnstore.xml";
     uint32_t    max_rows = 1000;
     Seconds     max_time = Seconds(5);
+    std::string debug;
```

Comment:
This is used as a boolean and should be declared as one. Extend the option processing if it doesn't support booleans.

Reply:
This is used as a string b/c librdkafka supports a [number of logging facilities](https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md) that could be combined.